### PR TITLE
Update README for why OpenVINO XAI &known issues

### PR DESCRIPTION
**Pull request by goodsong81**
* Add "why to use OpenVINO XAI"

image

* Update known issue as user warning for older version of OpenVINO
  - https://github.com/openvinotoolkit/openvino_xai/issues/26 is not reproduced from OpenVINO==2024.2
  - Reproducing test was added in https://github.com/openvinotoolkit/openvino_xai/pull/39

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,10 +81,13 @@ Most of CNNs and Transformer models from [Pytorch Image Models (timm)](https://g
 
 Please refer to the following known issues for unsupported models and reasons.
 
-* [OpenVINO IR branch insertion not working for models converted directly from torch models with OVC (#26)](https://github.com/openvinotoolkit/openvino_xai/issues/26)
 * [Runtime error from ONNX / OpenVINO IR models while conversion or inference for XAI (#29)](https://github.com/openvinotoolkit/openvino_xai/issues/29)
 * [Models not supported by white box XAI methods (#30)](https://github.com/openvinotoolkit/openvino_xai/issues/30)
 
+> **_WARNING:_**  OpenVINO XAI is fully validated on OpenVINO 2024.2.0. Following issue might be observed if used with older versions of OpenVINO.
```
Comment (GalyaZalesskaya) [inline README.md]:
```suggestion
> **_WARNING:_**  OpenVINO XAI is fully validated on OpenVINO 2024.2.0. Following issue might be observed if older the version of OpenVINO is used.
```

File: README.md [hunk 2]
```diff
@@ -30,6 +30,23 @@ Given **OpenVINO** models and input images, **OpenVINO XAI** generates **salienc
 which highlights regions of the interest in the inputs from the models' perspective
 to help users understand the reason why the complex AI models output such responses.
 
+Using this package, you can augment the model analysis & explanation feature
```
Comment (GalyaZalesskaya) [inline README.md:33]: It's just a suggestion, but what do you think about adding the actual "Why OpenVINO-XAI" heading here?

Comment (goodsong81) [reply README.md:33]: I don't think the "a few lines of code" is that much attractive for the heading.
I've just put a small "why" after the small "what".

File: README.md (hunk 1 above)
Comment (goodsong81) [reply README.md]: https://github.com/openvinotoolkit/openvino_xai/pull/41/commits/7229f36fa5d51cf9aabac7f249341f53068ef103

File: README.md (hunk 2 above)
Comment (negvet) [reply README.md:33]: We might probably need "Why OpenVINO-XAI" section somewhere in the readme.

But this new lines better be named "Where and how OV XAI". Although can be skipped from my perspective

**Review by negvet — APPROVED** [review]

**Review by GalyaZalesskaya — APPROVED** [review]

**Pull request merged**